app/Http/Livewire/Airfields/Index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Livewire\Airfields;
+
+use App\Http\Resources\UKCP\AirfieldCollection;
+use Illuminate\Support\Facades\Http;
+use Livewire\Component;
+
+class Index extends Component
+{
+    public string $search = '';
+
+    protected $queryString = ['search' => ['except' => ''], 'showEmpty' => ['except' => false]];
+
+    public bool $showEmpty = false;
+
+    public function render()
+    {
+        $url = $this->showEmpty ? '/admin/airfields?all=true' : '/admin/airfields';
+        $airfields = (new AirfieldCollection(Http::ukcp($url)['airfields']))->sortBy('code');
```

Comment:
See comment regarding using the macro - this feels quite difficult to read and understand what is going on.

Reply:
done